gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/testing/LocalPubsubHelper.java

```diff
@@ -35,63 +35,78 @@
  * A class that runs a Pubsub emulator instance for use in tests.
  */
 public class LocalPubsubHelper {
+
+  private final int port;
   private final LocalServiceHelper serviceHelper;
-  private final List<String> gcloudCommand;
-  private final URL emulatorUrl;
+
 
   // Local server settings
   private static final int DEFAULT_PORT = 8080;
   private static final String DEFAULT_HOST = "localhost";
+  private static final URL EMULATE_URL;
```

Comment:
Nit, I slightly prefer `EMULATOR_URL` (which is also in line with the previous name).